libs/test-common/src/main/java/org/codice/ddf/test/common/options/VmOptions.java

```diff
@@ -32,4 +34,41 @@ public static Option defaultVmOptions() {
         vmOption("-Djava.awt.headless=true"),
         vmOption("-Dfile.encoding=UTF8"));
   }
+
+  public static Option java9VmOptions() {
```

Comment:
Ah. I think this answers my question. We wrote this, not pax. It's the relevant settings that would apply to J9 or higher _because_ of the J9 changes. 

✏️   How about `java9andUpVmOptions()` instead? Not necessary.

Reply:
Maybe `javaModuleVmOptions()`

Reply:
Yes I like that a lot 💯